compiler-cli/src/main.rs

```diff
@@ -114,60 +117,112 @@ pub enum CliCommand {
         path: PathBuf,
     },
 
-    /// Output an executable. Defaults to 'a.out' in the current working
-    /// directory.
+    /// Compile to an ELF executable using libFIRM's backend.
+    /// Defaults to 'a.out' in the current working directory.
     #[structopt(name = "--compile-firm")]
     CompileFirm(CompileFirmOptions),
+
+    /// Compile to an ELF executable or molki assembly using the comprakt
+    /// backend. Defaults to 'a.out' in the current working directory.
+    #[structopt(name = "--compile")]
+    Compile(CompileOptions),
+}
+
+/// Backends supported by the [`CliCommand::Compile`] (`--compile`) command.
+#[derive(StructOpt, Debug, Clone)]
+pub enum CompileBackend {
+    #[structopt(name = "amd64")]
+    AMD64,
```

Comment:
This is called
```suggestion
    Amd64,
```

in the lowering branch. This should be consistent